packages/snfoundry/scripts-ts/deploy.ts

```diff
@@ -1,17 +1,65 @@
-import { deployContract, deployer, exportDeployments } from "./deploy-contract";
+// import { deployContract, deployer, exportDeployments } from "./deploy-contract";
 
-const deployScript = async (): Promise<void> => {
-  await deployContract(
-    {
-      owner: deployer.address, // the deployer address is the owner of the contract
-    },
-    "YourContract"
-  );
+// const deployScript = async (): Promise<void> => {
+//   await deployContract(
+//     {
+//       owner: deployer.address, // the deployer address is the owner of the contract
+//     },
+//     "YourContract"
+//   );
+// };
+
+// deployScript()
+//   .then(() => {
+//     exportDeployments();
+//     console.log("All Setup Done");
+//   })
+//   .catch(console.error);
+import {
+  deployContract,
+  deployer,
+  provider,
+  exportDeployments,
+} from "./deploy-contract";
+import { CallData, hash } from "starknet-dev";
+
+const checkWalletDeployed = async (walletAddress: string): Promise<boolean> => {
+  try {
+    const accountInfo = await provider.getAccount(walletAddress);
```

Comment:
Where is the `getAccount` from? This property does not exist on type `RpcProvider`

Reply:
i would rather change the `scripts-ts/deploy-contracts.ts` than the `deploy.ts` @raizo07 . The first handles all the main logic of the deploy so the error should be logged there.

Reply:
we refer to it as a whole @raizo07 , we will be more specific in the future. also dont forget to update your branch

Reply:
@raizo07 Hi mate. Any updates on this?